kqueen_ui/blueprints/ui/views.py

```diff
@@ -421,12 +455,14 @@ def handle(self):
                 msg = 'Could not send password reset e-mail, please try again later.'
                 logger.exception(msg)
                 flash(msg, 'danger')
-            flash('Password reset link was sent to your e-mail address.', 'success')
+            else:
```

Comment:
incorrect indent

Reply:
Vlad, it is ok, you can check it by viewing the whole file https://github.com/Mirantis/kqueen-ui/blob/6642220bc14a1457a251be82d1aca0454a7b037f/kqueen_ui/blueprints/ui/views.py

Reply:
oh sorry